numba/tests/test_bytes.py

```diff
@@ -0,0 +1,41 @@
+from numba.tests.support import (TestCase, no_pyobj_flags, MemoryLeakMixin)
+from numba import njit
+import unittest
+
+
+def addbytes_len_usecase(a, b):
+    output = len(b''.join([a, b]))
+    return output
+
+
+BYTES_EXAMPLES = [
```

Comment:
I would avoid making this a global since it doesn't necessary have use for all bytes tests. Instead perhaps making it a class variable of `TestBytesData` would be more appropriate.